packages/services/api/src/modules/preflight-script/module.graphql.ts

```diff
@@ -0,0 +1,53 @@
+import { gql } from 'graphql-modules';
+
+export const typeDefs = gql`
+  type PreflightScript {
+    id: ID!
+    sourceCode: String!
+    createdAt: DateTime!
+    updatedAt: DateTime!
+    # TODO: should I add this field?
+    # createdBy: User!
+  }
+
+  input CreatePreflightScriptInput {
+    sourceCode: String!
+  }
+
+  input UpdatePreflightScriptInput {
+    id: ID!
+    sourceCode: String!
+  }
+
+  extend type Mutation {
+    createPreflightScript(
+      selector: TargetSelectorInput!
+      input: CreatePreflightScriptInput!
+    ): PreflightScriptResult!
+    updatePreflightScript(
+      selector: TargetSelectorInput!
+      input: UpdatePreflightScriptInput!
+    ): PreflightScriptResult!
```

Comment:
Instead of having two mutation fields for creating and updating the preflight script we could instead just have a single one `Mutation.modifyPreflightScript`. Is there a specific reason for splitting creation and updating?

Reply:
> Is there a specific reason for splitting creation and updating?

just common practice of crud operations, changed to `updatePreflightScript` with SQL on conflict